src/components/CowPenContextMenu/CowPenContextMenu.js

```diff
@@ -197,24 +197,17 @@ export const CowPenContextMenu = ({
       </TabPanel>
       <TabPanel value={currentTab} index={2}>
         <ul className="card-list">
-          <li>
-            <Item
-              {...{
-                item: cowFeed,
-                isPurchaseView: true,
-                showQuantity: true,
-              }}
-            />
-          </li>
-          <li>
-            <Item
-              {...{
-                item: huggingMachine,
-                isPurchaseView: true,
-                showQuantity: true,
-              }}
-            />
-          </li>
+          {cowShopInventory.map((item, i) => (
+            <li key={i}>
```

Comment:
`key={i}` should work just fine here, but it seems easy enough to avoid an index-based key so I'll tweak that after merging this.

Reply:
This has been changed to `item.id` in https://github.com/jeremyckahn/farmhand/commit/0384b149f453486c2b38dcc0ffd8557d36513ddc.